src/engine/enginemaster.cpp

```diff
@@ -483,8 +483,12 @@ void EngineMaster::process(const int iBufferSize) {
     // m_masterGain takes care of applying the attenuation from
     // channel volume faders, crossfader, and talkover ducking.
     // Talkover is mixed in later according to the configured MicMonitorMode
+    double talkOvergain = 1.0;
+    if (!m_activeTalkoverChannels.isEmpty()) {
+        talkOvergain = m_pTalkoverDucking->getGain();
+    }
```

Comment:
m_activeTalkoverChannels is cleared and then populated in processChannels() which is called a few lines above, so I guess it's correct to use it for checking if any talkover is enabled?